lib/chewy/index/observe/callback.rb

```diff
@@ -0,0 +1,34 @@
+module Chewy
+  class Index
+    module Observe
+      class Callback
+        def initialize(executable, filters = {})
+          @executable = executable
+          @if_filter = filters[:if]
+          @unless_filter = filters[:unless]
+        end
+
+        def call(context)
+          return if !@if_filter.nil? && !eval_filter(@if_filter, context)
+          return if !@unless_filter.nil? && eval_filter(@unless_filter, context)
```

Comment:
how about adding 
```
when nil
  true
```
in `eval_filter`? 
Then this could become
```suggestion
          return unless eval_filter(@if_filter, context)
          return if eval_filter(@unless_filter, context)
```
whih is nicely symmetrical :D

Reply:
argh, you're right of course